testsuite/tests/kuadrant/authorino/caching/metadata/test_caching.py

```diff
@@ -16,7 +16,7 @@ def cache_ttl():
 @pytest.fixture(scope="module")
 def authorization(authorization, module_label, expectation_path, cache_ttl):
     """Adds Cached Metadata to the AuthConfig"""
-    meta_cache = Cache(cache_ttl, Value(jsonPath="context.request.http.path"))
+    meta_cache = Cache(cache_ttl, ValueDynamic(valueFrom="context.request.http.path"))
```

Comment:
As these `Value` dataclasses require only one value and class names are self-explanatory, I'd use positional arguments on instances.